lib/auth/grpcserver.go

```diff
@@ -4500,6 +4500,44 @@ func (g *GRPCServer) SetInstaller(ctx context.Context, req *types.InstallerV1) (
 	return &emptypb.Empty{}, nil
 }
 
+func (g *GRPCServer) SetUIConfig(ctx context.Context, req *types.UIConfigV1) (*emptypb.Empty, error) {
+	auth, err := g.authenticate(ctx)
```

Comment:
Doesn't need to be in this PR, but I would consider leaving yourself a TODO to emit an audit event when the UI config is updated or deleted.